solvcon/parcel/gas/inout.py

```diff
@@ -65,12 +65,18 @@ def __init__(self, cse, show_bclist=False, perffn=None, **kw):
         super(MeshInfoHook, self).__init__(cse, **kw)
 
     def preloop(self):
+        flag_generic_bc = False
```

Comment:
Do you mean that your comments were gone in the "Files changed" session? I think that is because I merged each fix-up into its related commit. "git rebase -i" feature was used and then I merged the fixup into the previous commit to make the commit history look better. This action changes the UUID of the commit, so Github regards it as a new commit, and only shows the "files changed" comments and code change for the latest commits.

From my perspective, it's ok to me because your comments are reserved in this conversation session. Just to click "show outdated" to expand the old comments.